### PR TITLE
164859698 layout the boilerplate

### PR DESCRIPTION
**Pull request by willeswa**
### What does this PR do?

Set up the project boilerplate

### Description of Task to be completed?

[x] Use create-react-app to bootstrap the layout
[x] Install the Airbnb as the default linting tool

### How should this be manually tested?

Access the repository on GitHub from [here](https://github.com/deferral/ah-django-frontend.git)
Switch to the branch ch-setup-boilerplate-164859698
You should see a directory called app with all the setup files

### Any background context you want to provide?

The create-react-app has the essentials that sets up the basis of developing the frontend app

### What are the relevant pivotal tracker stories?

[#164859698](https://www.pivotaltracker.com/story/show/164859698)

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```
Comment (philophilo) [inline app/README.md]: This readme should be in the root directory

File: app/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (philophilo) [inline app/public/manifest.json]: Change the names here

File: app/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+
+### Deployment
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+
+### `npm run build` fails to minify
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Comment (philophilo) [inline app/README.md]: Get rid of the parts that are not important have the project running. Generic information regarding React is not important

File: package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "extends": "react-app"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not ie <= 11",
+    "not op_mini all"
+  ],
+  "devDependencies": {
+    "eslint": "^5.3.0",
```
Comment (wangonya) [inline package.json]: ~My system is complaining that this version of eslint is a bit ancient. It requires `5.12.0`. Any reason you chose to go with this? Or was it the default?~

~Never mind. The problem was local.~

Comment (wangonya) [reply package.json]: After fixing my local settings, I realized npm was still giving me warnings. We are having two different versions of `eslint` installed on the same project. I don't know if everything is fine on your side, but things are not working well on mine with both. I will remove the lower version.

Screenshot 2019-04-16 at 17 19 27

File: README.md [hunk 5]
```diff
@@ -1 +1,18 @@
-# ah-django-frontend
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
```
Comment (philophilo) [inline README.md:4]: Use your own words to describe your project, how to run it... 
what do you run to start the project... Make it easy for a 5-year-old 😁

**Pull request merged**